### ISSUE TITLE
Support AWS credentials in environment variables

### ISSUE THREAD

**Issue by mbarneyjr** [Theodo-UK/sls-dev-tools#34]
If you have your AWS credentials exported to environment variables, and not in a profile, you will get the error:

```
CredentialError: Missing credentials in config, if using AWS_CONFIG_FILE, set AWS_SDK_LOAD_CONFIG=1
```

**To Reproduce**
1. Export your aws credentials to your environment variables, making sure you don't have a default profile
2. Run the tool against a serverless stack

**Expected behavior**
It should function normally

**Screenshots**
![image](https://user-images.githubusercontent.com/11701776/77545933-89cd6f80-6e81-11ea-870b-04e8fc96f3fc.png)

**If relevant**
 - OS: OS X
 - Deployment Framework [SAM]

**Comment by BenEllerby** [comment]
Thanks @mbarneyjr this will be resolved by PR #35 that @DotGav has opened. Should be released tomorrow!

Any other feedback & feature requests very welcome!

**Comment by BenEllerby** [comment]
@mbarneyjr update to version 0.1.3 and this will be resolved :) 

Thank @DotGav for the great work!

Please raise more issues, it helps us a lot!
